src/gui/properties/pieceavailabilitybar.cpp

```diff
@@ -44,7 +44,7 @@ QVector<float> PieceAvailabilityBar::intToFloatVector(const QVector<int> &vecin,
     QVector<float> result(reqSize, 0.0);
     if (vecin.isEmpty()) return result;
 
-    const float ratio = vecin.size() / (float)reqSize;
+    const float ratio = vecin.size() / static_cast<float>(reqSize);
```

Comment:
do you mind move the casting to `vecin.size()`?
put casting up front is easier to read & have more consistence IMO.